java-extensions/jni-connector/src/main/java/com/starrocks/jni/connector/TypeMapping.java

```diff
@@ -21,14 +21,16 @@ public class TypeMapping {
     public static Map<String, OffHeapColumnVector.OffHeapColumnType> hiveTypeMappings = new HashMap<>();
     static {
         hiveTypeMappings.put("byte", OffHeapColumnVector.OffHeapColumnType.BYTE);
-        hiveTypeMappings.put("bool", OffHeapColumnVector.OffHeapColumnType.BOOLEAN);
```

Comment:
do you want to add one  instead of rename it?

Reply:
just rename，original name is not correct